guides/runbooks/orca-zombie-executions.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: single
+title:  "Orca: Zombie Executions"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+Aliases: `orphaned execution`
+
+A pipeline or task is "stuck" in a `RUNNING` state but not doing anything.
+
+A zombie Execution is one that has a `status` in the database as `RUNNING` but there are no messages in Orca's work queue or unacked set. 
+In this scenario, Orca will never complete the work. 
+
+# Diagnosis
+
+Logs will be emitted regularly for Executions that are currently running in Orca via the `QueueProcessor` class, which will look similar to the following example.
```

Comment:
@robzienert I notice you've got all instances of "Execution/s" capitalized. I *think* it doesn't need to be. But feel free to set me straight.

Reply:
I was capitalizing it to distinguish between the model Execution (representing either pipelines or orchestrations) versus an execution of a pipeline or orchestration. Happy to change it.

Reply:
Nah, it's fine